src/app/pages/viewing/messages/messages.component.ts

```diff
@@ -13,11 +16,50 @@ import { Fonts } from '../../../shared/interfaces';
 export class MessagesComponent implements OnInit {
   fonts: Fonts;
   messageToDelete: string;
-
-  constructor(public viewService: ViewingService, public authService: AuthService) { }
+  constructor(public viewService: ViewingService, public authService: AuthService, private toastr: ToastrService) { }
 
   ngOnInit(): void {
     this.fonts = FontService.fonts;
+    this.viewService.nowMillis = Timestamp.now().toMillis();
+  }
+
+  /**
+   * Generate the lightness value of HSL from RBG
+   */
+  getLightness(r, g, b) {
+    r /= 255, g /= 255, b /= 255;
```

Comment:
"Reports comma expressions. Comma expressions are often a sign of overly clever code, and may lead to subtle bugs. Comma expressions in the initializer or update section of a for loop are ignored by this inspection."
```suggestion
    r /= 255;
    g /= 255;
    b /= 255;
```